portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -72,7 +94,7 @@ private String convertToJson(ArrayList<Comment> list) {
    */
   private String getParameter(HttpServletRequest request, String name, String defaultValue) {
     String value = request.getParameter(name);
-    if (value == null) {
+    if (value.isEmpty()) {
```

Comment:
I would recommend adding a null check before using isEmpty as the `request.getParameter` does not seem to specify a default if 'name' is not found

Reply:
Does what I've changed it to look better?